hudi-common/src/test/java/org/apache/hudi/common/testutils/FileCreateUtils.java

```diff
@@ -101,7 +101,11 @@ public static String markerFileName(String instantTime, String fileId, IOType io
   }
 
   public static String markerFileName(String instantTime, String fileId, IOType ioType, String fileExtension) {
-    return String.format("%s_%s_%s%s%s.%s", fileId, WRITE_TOKEN, instantTime, fileExtension, HoodieTableMetaClient.MARKER_EXTN, ioType);
+    return markerFileName(instantTime, fileId, ioType, fileExtension, WRITE_TOKEN);
+  }
+
+  public static String markerFileName(String instantTime, String fileId, IOType ioType, String fileExtension, String writeToken) {
+    return String.format("%s_%s_%s%s%s.%s", fileId, writeToken, instantTime, fileExtension, HoodieTableMetaClient.MARKER_EXTN, ioType);
```

Comment:
Usually we do not add logic only for testing purpose.

Reply:
Will implement a custom marker creation logic within the test then

Reply:
@danny0405 ignore my last message. 

May i ask why it is not recommended to add overloaded static functions in test classes that is mainly used for testing? 

The file i am modifying is in the project<>namespace that belongs to the test scope.

I feel it is entirely reasonable to add helper functions to a helper class for testing purposes. WDYT? 

hudi-common/src/**test**/java/org/apache/hudi/common/testutils/FileCreateUtils.java

CMIIW, thanks.

Reply:
Yeah, you are right, my mistake